src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        this.radius = radius;
+    }
+
+    @Override
```

Comment:
The getArea method is implemented correctly and follows the single responsibility principle.